.github/workflows/tests.yml

```diff
@@ -40,19 +40,69 @@ jobs:
                 image: rabbitmq:3.8.3
                 ports:
                     - 5672:5672
+            mongodb:
+                image: mongo
+                ports:
+                    - 27017:27017
+            couchbase:
+                image: couchbase:6.5.1
+                ports:
+                    - 8091:8091
+                    - 8092:8092
+                    - 8093:8093
+                    - 8094:8094
+                    - 11210:11210
+            sqs:
+                image: asyncaws/testing-sqs
```

Comment:
have you considered https://github.com/localstack/localstack?

Reply:
No, I'm mainly migrating what was there on Travis.

Reply:
aha, fair.

Reply:
Last time we tried (@Nyholm and I) the image did not match our expectations:
- does not supports all features (MessageAttributes used to store `messenger headers`)
- Api contract is not ISO with AWS (mostly format of errors/exception)

But we may have mistaken our implementation, if you know `localstack` well, up for a PR?

Reply:
We use it, but haven't not encountered those issues. Your impl is probably a lot lower level than ours, if you're talking about `async-aws`!